### PR TITLE
French translation

### LINKED ISSUE #40: Typos & problems with Fr location
Introduction in English
image

and anomalous "les" 
image

will do a PR when I have time

### PR DESCRIPTION
**Pull request by Prometheos2**
Fixes #40 again
Works on #27's 2fe8f61 commit

### CODE REVIEW

File: assets/texts/fr/amoDescription.html [hunk 1]
```diff
@@ -11,6 +11,9 @@
 Il est basé sur le formidable projet <a href="https://missive.github.io/emoji-mart/">emoji-mart</a>. Vous pouvez déjà tester de nombreuses fonctionnalités sur le site, sans installer l'extension.
 Elle ne fonctionne que sur Firefox v.63 ou plus récent.
 
+Il a également une recherche intégrée, donc vous pouvez l'utiliser comme une "fiche de révision" d'émojis. <b>Trouvez
```
Comment (rugk) [inline assets/texts/fr/amoDescription.html]: Strange line wrapping (again?)…

File: assets/texts/fr/amoDescription.html [hunk 2]
```diff
@@ -52,6 +57,10 @@
 Ce module complémentaire est un logiciel gratuit, libre et "open-source" et est dévelopé sur GitHub. <a href="https://github.com/rugk/awesome-emoji-picker">Forker le sur GitHub</a> et contribuez.
 <a href="https://github.com/rugk/awesome-emoji-picker/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22">Voici des tâches faciles à gèrer pour commencer.</a>
 
+Le développement de ce module a aussi été provoqué, car le développement d'un autre module nommé <a
+	href="https://github.com/johannhof/emoji-helper/issues/127">“Emoji Helper” ou “Emoji Cheatsheet” est en suspend</a>.
```
Comment (rugk) [inline assets/texts/fr/amoDescription.html]: line wrapping

File: assets/texts/fr/amoDescription.html (hunk 1 above)
Comment (Prometheos2) [reply assets/texts/fr/amoDescription.html]: Argh, my bad.
I forgot to disable the code reformat at commit.

**Comment by rugk** [comment]
Uhm, sorry for the late reaction. Is everything resolved here? Can this be merged?

**Comment by Prometheos2** [comment]
Unless I've missed an update on languages, that should be good.

**Comment by rugk** [comment]
Okay, so thanks. It's merged and deployed on AMO.

**Pull request merged**